### PR TITLE
Goebaht

### PR DESCRIPTION
**Pull request by ScottSWu**
Changes to Servo, adding HBridge

### CODE REVIEW

File: minibot/hardware/gpio.py [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
+    """
+    def __init__(self, pin):
+        """
+        Constructor.
+        Args:
+            pin (int): Digital pin number.
+        """
+        self.pin = pin
+
+    def set_low(self):
```
Comment (chynu) [inline minibot/hardware/gpio.py:37]: Documentation

File: minibot/hardware/gpio.py [hunk 2]
```diff
@@ -2,6 +2,44 @@
...
+        raise NotImplementedError
+
+    def set_high(self):
```
Comment (chynu) [inline minibot/hardware/gpio.py]: Documentation

File: minibot/hardware/rpi/gpio.py [hunk 3]
```diff
@@ -3,27 +3,59 @@
...
+    """
+    Read digital input from the pin.
+    Return:
+        int: 0 or 1 for LOW or HIGH voltage.
+    """
+    def read(self):
+        return RGPIO.input(self.pin)
+
+class DigitalOutput(MDigitalOutput):
+    def __init__(self, pin):
```
Comment (chynu) [inline minibot/hardware/rpi/gpio.py]: Documentation, here and below.

**Review by chynu — APPROVED** [review]
bup-gtm

**Comment by ScottSWu** [comment]
im bup

**Comment by ScottSWu** [comment]
buppity

**Pull request merged**